g3doc/LangRef.md

```diff
@@ -756,36 +756,58 @@ Examples of memref static type
 
 ```mlir {.mlir}
 // Identity index/layout map
-#imapA = (d0, d1) -> (d0, d1) size (16, 32)
+#imapA = (d0, d1) -> (d0, d1)
 
 // Column major layout.
-#imapB = (d0, d1, d2) [s0] -> (d2, d1, d0) size (s0, 4, 16)
+#imapB = (d0, d1, d2) -> (d2, d1, d0)
+
+// A 2-d tiled layout with tiles of size 128 x 256.
+#imapC = (d0, d1) -> (d0 div 128, d1 div 256, d0 mod 128, d0 mod 256)
+
+// A tiled data layout with non-constant tile sizes.
+#imapD = (d0, d1)[s0, s1] -> (d0 floordiv s0, d1 floordiv s1,
+                              d0 mod s0, d1 mod s1)
+
+// A layout that yields a padding on two at either end of the minor dimension.
+#imapE = (d0, d1) -> (d0, (d1 + 2) floordiv 2, (d1 + 2) mod 2)
```

Comment:
What about a semantic name for these? `#tile2d_128x256`, `#tile2d_dynamic`, `#padded`...

Reply:
Done, thanks.